Importer/CsvImporter.php

```diff
@@ -183,7 +170,7 @@ protected function importData(array $parsedData, $dataIsAssociativeArray, $flush
             return;
         }
 
-        if (!$flush || $this->log->containErrors()) {
+        if (!$flush || $this->log->getErrorCount() > 0) {
```

Comment:
somehow i thought `containErrors()` was nicely readable. was there a necessary reason for this change?